Files/App.xaml

```diff
@@ -17,20 +17,23 @@
                             <SolidColorBrush x:Key="CloudDriveSyncStatusExcludedColor" Color="#AAAAAA" />
                             <SolidColorBrush x:Key="ApplicationPageBackgroundThemeBrush" Color="White" />
                             <SolidColorBrush x:Key="SystemControlPageBackgroundMediumAltMediumBrush" Color="#99000000" />
+                            <SolidColorBrush x:Key="GrayTextColor" Color="#99000000" />
```

Comment:
Instead of providing alpha value in color itself try something like this:
`<SolidColorBrush x:Key="GrayTextColor" Color="#000000" Opacity="0.6"/>`

Reply:
tested, still not working